corefoundation.gemspec

```diff
@@ -19,9 +21,9 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.6'
 
-  s.add_runtime_dependency "ffi"
-  s.add_development_dependency "rspec", "~>2.10"
-  s.add_development_dependency "rake"
-  s.add_development_dependency "yard"
-  s.add_development_dependency "redcarpet"
+  s.add_runtime_dependency 'ffi'
+  s.add_development_dependency 'rake'
```

Comment:
if this is 3.x we should make it ">= 3"

Reply:
I also pinned `ffi` to `1.15.x` along with it in https://github.com/chef/corefoundation/pull/10/commits/bdeddf5c5b893f9417c03e757a1f1c566ede9c96.